src/query/helper.rs

```diff
@@ -532,3 +555,32 @@ pub(crate) fn unpack_table_alias(table_ref: &TableRef) -> Option<DynIden> {
         | TableRef::DatabaseSchemaTableAlias(_, _, _, alias) => Some(SeaRc::clone(alias)),
     }
 }
+
+pub(crate) fn cast_enum_as_text<C>(expr: Expr, col: &C) -> SimpleExpr
+where
+    C: ColumnTrait,
+{
+    cast_enum_text_inner(expr, col, |col, _| col.as_enum(Alias::new("text")))
```

Comment:
Is it possible to use a const, i.e. `#[derive(Iden)] struct Text` ?

Reply:
Sure!